browser/src/VariantList/Variants.tsx

```diff
@@ -184,7 +192,7 @@ const Variants = ({
     }
 
     index = renderedVariants.findIndex(
-      (variant: any, i: any) =>
+      (variant: Variant, i: any) =>
```

Comment:
Lots of type comments on this one, apologies. I think we can narrow down the type of `i` to `number`.